### ISSUE TITLE
"Dashboards" button stopped working in Teslamate 1.28.1

### ISSUE THREAD

**Issue by oleglypkan** [teslamate-org/teslamate#3502]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

After upgrading to Teslamate 1.28.1, "Dashboards" button that showed the list of available dashboards stopped working. Nothing happens when I click it. When I revert Grafana image to teslamate/grafana:1.27.4, it starts working again.

### Expected Behavior

Clicking "Dashboards" button should show the list of available dashboards.

### Steps To Reproduce

Have Teslatemate 1.28.1 installed.
Open any Grafana dashboard, e.g. Overview.
Click "Dashboards" button in the Overview dashboard.
Problem: nothing happens after clicking the button, the list of dashboards is not displayed.

### Relevant log output

```shell
I do not see any relevant errors in Grafana logs:

teslamate-grafana-1  | logger=context userId=1 orgId=1 uname=teslamate t=2023-11-26T18:29:19.336345623Z level=info msg="Request Completed" method=GET path=/api/live/ws status=400 remote_addr=192.168.0.1 time_ms=8 duration=8.192778ms size=12 referer= handler=/api/live/ws
teslamate-grafana-1  | logger=context userId=1 orgId=1 uname=teslamate t=2023-11-26T18:29:28.836466057Z level=info msg="Request Completed" method=GET path=/api/live/ws status=400 remote_addr=192.168.0.1 time_ms=5 duration=5.404104ms size=12 referer= handler=/api/live/ws
```

### Screenshots

teslamate/grafana:1.28.1

![Screenshot from 2023-11-26 10-17-44](https://github.com/teslamate-org/teslamate/assets/16422686/80a1ea81-294f-4896-a38a-ee5d577f9145)

teslamate/grafana:1.27.4

![Screenshot from 2023-11-26 10-16-20](https://github.com/teslamate-org/teslamate/assets/16422686/690c6578-e85c-4370-870e-3aa768190f6b)

### Type of installation

Docker

### Version

v1.28.1

**Comment by SiNaPsEr0x** [comment]
Sorry but don't mark fixed if it hasn't been fixed in any way ...

**Comment by JakobLichterfeld** [comment]
Fixed with v1.28.2
